pff_equipment_testing/src/pff_equipment_testing/install/redboard.rules

```diff
@@ -0,0 +1 @@
+ACTION=="add" SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6015", MODE="0666", GROUP="dialout"
```

Comment:
add a symlink to your rules `SYMLINK+="/pff/redboard"` then your device paths in your yaml won't change based on the order they are discovered. it will always be the same path

Reply:
i would do this for the force gage too

Reply:
I'll go in and change this when Andy is done, I'll need the hardware to get vendor/product IDs